pkg/codegen/internal/test/testdata/types.json

```diff
@@ -0,0 +1,224 @@
+                "go": {
+                  "input": "pulumi.IntPtrInput",
+                  "plain": "*int"
+                },
+                "nodejs": {
+                  "input": "pulumi.Input\u003cnumber\u003e | undefined",
+                  "plain": "number | undefined"
+                },
+                "python": {
+                  "input": "Optional[pulumi.Input[int]]",
+                  "plain": "Optional[int]"
+                }
+              }
+            }
+          }
+        },
+        "number": {
+          "type": "number",
+          "language": {
+            "test": {
+              "expected": {
+                "dotnet": {
+                  "input": "Input\u003cdouble\u003e?",
+                  "plain": "double?"
+                },
+                "go": {
+                  "input": "pulumi.Float64PtrInput",
+                  "plain": "*float64"
+                },
+                "nodejs": {
+                  "input": "pulumi.Input\u003cnumber\u003e | undefined",
+                  "plain": "number | undefined"
+                },
+                "python": {
+                  "input": "Optional[pulumi.Input[float]]",
+                  "plain": "Optional[float]"
+                }
+              }
+            }
+          }
+        },
+        "requiredBoolean": {
+          "type": "boolean",
+          "language": {
+            "test": {
+              "expected": {
+                "dotnet": {
+                  "input": "Input\u003cbool\u003e",
+                  "plain": "bool"
+                },
+                "go": {
+                  "input": "pulumi.BoolInput",
+                  "plain": "bool"
+                },
+                "nodejs": {
+                  "input": "pulumi.Input\u003cboolean\u003e",
+                  "plain": "boolean"
+                },
+                "python": {
+                  "input": "pulumi.Input[bool]",
+                  "plain": "bool"
+                }
+              }
+            }
+          }
+        },
+        "requiredInteger": {
+          "type": "integer",
+          "language": {
+            "test": {
+              "expected": {
+                "dotnet": {
+                  "input": "Input\u003cint\u003e",
+                  "plain": "int"
+                },
+                "go": {
+                  "input": "pulumi.IntInput",
+                  "plain": "int"
+                },
+                "nodejs": {
+                  "input": "pulumi.Input\u003cnumber\u003e",
+                  "plain": "number"
+                },
+                "python": {
+                  "input": "pulumi.Input[int]",
+                  "plain": "int"
+                }
+              }
+            }
+          }
+        },
+        "requiredNumber": {
+          "type": "number",
+          "language": {
+            "test": {
+              "expected": {
+                "dotnet": {
+                  "input": "Input\u003cdouble\u003e",
```

Comment:
This can be `<` and '>' I think the JSON parser will read it just fine.

Reply:
The escaping was performed automatically by the JSON marshaller--I'll see if I can disable it.

Reply:
Perhaps not worth it, just aesthetic.

Reply:
For reference, see https://golang.org/pkg/encoding/json/#HTMLEscape

I was able to disable the escaping.